packages/firebase_remote_config/firebase_remote_config/ios/Classes/FLTFirebaseRemoteConfigPlugin.m

```diff
@@ -0,0 +1,266 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Firebase/Firebase.h>
+#import <firebase_core/FLTFirebasePluginRegistry.h>
+
+#import "FLTFirebaseRemoteConfigPlugin.h"
+#import "FLTFirebaseRemoteConfigUtils.h"
+
+NSString *const kFirebaseRemoteConfigChannelName = @"plugins.flutter.io/firebase_remote_config";
+
+@interface FLTFirebaseRemoteConfigPlugin ()
+@property(nonatomic, retain) FlutterMethodChannel *channel;
```

Comment:
this field is unused & could be removed.

It only seems to be set to nil in `dealloc`

@kroikie @rrousselGit